package.json

```
@@ -66,6 +66,21 @@
   },
   "resolutions": {
     "@openshift-console/plugin-shared/showdown/yargs/cliui/wrap-ansi/strip-ansi/ansi-regex": "2.0.0",
+    "**/@babel/core/semver": "6.3.1",
+    "**/@babel/helper-compilation-targets/semver": "6.3.1",
+    "**/@openshift-console/dynamic-plugin-sdk-webpack/semver": "6.3.1",
+    "**/@typescript-eslint/typescript-estree/semver": "7.7.1",
+    "**/cypress/semver": "7.7.1",
+    "**/eslint-config-airbnb-base/semver": "6.3.1",
+    "**/eslint-plugin-import/semver": "6.3.1",
+    "**/eslint-plugin-react/semver": "6.3.1",
+    "**/fork-ts-checker-webpack-plugin/semver": "7.7.1",
+    "**/getos/async": "3.2.4",
+    "**/istanbul-lib-instrument/semver": "7.7.1",
+    "**/make-dir/semver": "6.3.1",
+    "**/normalize-package-data/semver": "6.3.1",
+    "**/ts-loader/semver": "7.7.1",
```

why do u need so many conditions ?
```suggestion
    "semver": "^6.3.1",
    "**/@typescript-eslint/typescript-estree/semver": "7.7.1",
    "**/cypress/semver": "7.7.1",
    "**/getos/async": "3.2.4",
    "**/istanbul-lib-instrument/semver": "7.7.1",
    "**/ts-loader/semver": "7.7.1",
```

---

Sure will do this